.factory/automation.yml

```diff
@@ -108,7 +108,8 @@ build:
         - build
         - build-dependency
         - test-java
-        - test-rust
+        - test-rust-unit
+        - test-rust-behaviour
```

Comment:
I'd make the jobs expected to fail explicitly commented out with todos, like in server. It would helps us understand the scope of updates for the future & start keeping the build green in 3.0

Reply:
I'm afraid we might forget. So I'm happy for behaviour to be red.